src/styles/search-bar.css

```diff
@@ -0,0 +1,109 @@
+/* #region search-bar */
+.search-bar {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: var(--font-color);
+
+  margin: 20px auto 0;
+}
+
+.search-bar--big {
+  height: 70px;
+  top: 20px;
+}
+
+.search-bar--small {
+  top: 110px;
+  height: 42px;
+}
+
+/* #endregion */
+
+/* #region content-wrapper */
+.search-bar__content-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+/* #endregion */
+
+/* #region icon (content-wrapper::after) */
+.search-bar__content-wrapper::after {
+  position: absolute;
+  content: '';
+  display: block;
+
+  background-image: url('../images/Search.svg');
```

Comment:
Ensure that the image file `Search.svg` exists in the `../images/` directory and is accessible. This will prevent issues with the background image not displaying.